src/lib/es6.core.d.ts

```diff
@@ -0,0 +1,587 @@
+declare type PropertyKey = string | number | symbol;
+
+interface Symbol {

```

Comment:
we can split symbols to an es6.symbols.d.ts, and the rest to an es6.iterables.d.ts